config/database.yml

```diff
@@ -0,0 +1,22 @@
+default: &default
```

Comment:
1. Open `.gitignore` and add `config/database.yml`
2. Run command: `git rm --cached config/database.yml`